core/src/test/java/net/sourceforge/jnlp/TinyHttpdImpl.java

```diff
@@ -136,8 +135,8 @@ public void run() {
                     String request = t.nextToken();
                     String filePath = t.nextToken();
 
-                    boolean isHeadRequest = request.equals(ResourceTracker.RequestMethods.HEAD.toString());
-                    boolean isGetRequest = request.equals(ResourceTracker.RequestMethods.GET.toString());
+                    boolean isHeadRequest = request.equals(HttpMethod.HEAD.name());
+                    boolean isGetRequest = request.equals(HttpMethod.GET.name());
```

Comment:
Here is can see some problems by using enum.name():
- lower to upper case
- refactoring of enum may break this code.

Maybe we can define a String name in the enum class:

```
enum MyEnum {
A("1"), B("2");

private final String name;

public MyEnum(final String name) { this.name = name;}

public String getName() {return name;}

}
```

Reply:
I would not call this method getName() as it is too close to name() and the name of the method does not give a hint when to use which method.

Maybe use `request.equalsIgnoreCase(HttoMethod.GET.name())` I think it is safe to assume that the Enum HttpMethod will contain valid Http method names